kinto/__main__.py

```diff
@@ -11,7 +11,7 @@
 from kinto import __version__
 from kinto.config import init
 
-DEFAULT_CONFIG_FILE = 'config/kinto.ini'
+DEFAULT_CONFIG_FILE = os.getenv('KINTO_INI', 'config/kinto.ini')
```

Comment:
Looks like you are also fixing #1339

Reply:
You are right I should do that in different PR's

Reply:
Not necessarily, those are tiny fixes, but don't forget to update CHANGELOG